Runtime/Scripts/PeriodicSnapshots.cs

```diff
@@ -134,12 +142,13 @@ private IEnumerator TakeSnapshotsAcrossFrames(string timestamp, string path)
 
         private static string FetchArchivePath(string timestamp)
         {
-            return $"{Application.persistentDataPath}{Path.DirectorySeparatorChar}snapshot-series-{timestamp}.zip";
+            return $"{Application.persistentDataPath}{Path.DirectorySeparatorChar}{timestamp}-schaduwstudie.zip";
+            //2025-03-04T15-51-schaduwstudie.zip
         }
 
         private static string FetchPath(string timestamp)
         {
-            string path = $"{Application.persistentDataPath}{Path.DirectorySeparatorChar}snapshot-series-{timestamp}";
+            string path = $"{Application.persistentDataPath}{Path.DirectorySeparatorChar}{timestamp}-schaduwstudie";
```

Comment:
schaduwstudie is niet relevant voor deze library, met deze tekst is de library niet meer inzetbaar voor andere redenen waarom je meerdere snapshot wil maken

Reply:
Tevens: duplicatie van de filename hier